docs/content/integrations/dbt/reference.mdx

```diff
@@ -304,6 +306,40 @@ def my_dbt_assets(context: OpExecutionContext, dbt: DbtCliResource):
     yield from dbt.cli(["build"], context=context).stream()
 ```
 
+### Customizing Auto-Materialize Policies
+
+For dbt models, seeds, and snapshots, the default <PyObject object="AutoMaterializePolicy"/> will be `None`.
+
+To add an AutoMaterializePolicy to a dbt node, you can define a `meta` key in your dbt project file, on your dbt node's property file, or on the node's in-file config block. This policy may be one of two types, `"eager"` or `"lazy"`. The following example provides an eager AutoMaterializePolicy for the following model:
+
+```yaml
+models:
+  - name: customers
+    config:
+      meta:
+        dagster:
+          auto_materialize_policy:
+            type: eager
+```
+
+### Customizing Freshness Policies
+
+For dbt models, seeds, and snapshots, the default <PyObject object="FreshnessPolicy"/> will be `None`.
+
+To add a FreshnessPolicy to a dbt node, you can define a `meta` key in your dbt project file, on your dbt node's property file, or on the node's in-file config block. This config accepts identical arguments to the <PyObject object="FreshnessPolicy"/> class. The following example applies a FreshnessPolicy for the following model:
```

Comment:
```suggestion
To add a `FreshnessPolicy` to a dbt node, you can define a `meta` key in your dbt project file, on your dbt node's property file, or on the node's in-file config block. This config accepts identical arguments to the <PyObject object="FreshnessPolicy"/> class. The following example applies a `FreshnessPolicy` for the following model:
```

Also, why does the second instance of `FreshnessPolicy` get a `PyObject`, but not the others? As you use it in the first sentence of this section, I don't think it's needed in this paragraph tbh